src/test/java/teammates/test/cases/browsertests/StudentHomePageUiTest.java

```diff
@@ -42,7 +41,8 @@ protected void prepareTestData() {
         FeedbackSessionAttributes gracedFeedbackSession =
                 BackDoor.getFeedbackSession("SHomeUiT.CS2104", "Graced Feedback Session");
         // TODO: change to actual now once HTML time zone detection is fixed
```

Comment:
Have you document it somewhere, e.g. issues in case we forget this later?

Reply:
Just did - #8640.